packages/storybook/.storybook/style.scss

```diff
@@ -33,4 +33,17 @@ a.sbdocs-a:hover {
 
 .resize-y {
   resize: vertical;
+}
+
+/** Global styles for USWDS 3 integration into Storybook **/
+.usa-label {
+  font-size: 15px!important;
+}
+
+button {
+  padding: 10px 20px!important;
+}
+
+h1 {
+  font-size: 32px!important;
```

Comment:
Can you link to the USWDS3 styles that we are mimicking here?

Reply:
Well now I don't see any difference when I remove it so I guess I can remove it from here.

But the Formation rem style for H1 is coming from here (`h1: 4rem`):
https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/utilities/_font-size.scss#L8

And in USWDS, they seem to be setting it with the type scale token "14" which I think is the equivalent of 40px (?). This is the trail I followed to find that:

1. https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/mixins/typography/headings.scss#L16-L17

```
@mixin h1 {
  @include typeset("heading", $theme-h1-font-size, $theme-heading-line-height);
  font-weight: fw("bold");
}
```

2. https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/settings/_settings-typography.scss#L400

```
$theme-h1-font-size: "2xl" !default;
```

3. https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/variables/type-scale.scss#L14

```
"2xl": system-type-scale(settings.$theme-type-scale-2xl),
```

4. https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/settings/_settings-typography.scss#L308

```
$theme-type-scale-2xl: 14 !default;
```

4. https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/settings/_settings-typography.scss#L354

```
14:         40px
```